workshops/models.py

```diff
@@ -227,9 +228,14 @@ def ongoing_events(self):
         return queryset
 
     def unpublished_events(self):
-        '''Return a queryset for events that are not yet published.'''
+        '''Return events without URLs that are upcoming or have unknown starts.
```

Comment:
On Sat, Jun 13, 2015 at 11:11:45AM -0700, Greg Wilson wrote:
> > @@ -227,9 +228,14 @@ def ongoing_events(self):
> >          return queryset
> >  
> >      def unpublished_events(self):
> > -        '''Return a queryset for events that are not yet published.'''
> > +        '''Return events without URLs that are upcoming or have unknown starts.
> 
> It might make more sense to change the name from "unpublished" to
> "incomplete" to better reflect the actual semantics. If so, should I
> put that into this PR or do it separately?

I dunno.  Unpublished has a clear consequence (“is it going to show up
on our public event list or not?”).  What's “incomplete” about?  For
example, we may want to show public information before we've locked in
the full complement of instructors.  That sounds like “published but
incomplete”.  So I'm unconvinced on the name change.

As for whether that lands in this PR or another, I don't really mind.
If we expect one issue (the logic change or the rename) to stick in
discussion for longer, it's better to split them apart (so the
non-controversial one can go through).  In this case, I expect us to
reach consensus on both the logic change and the rename quickly, so it
shouldn't matter.